docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+# Use postgres/example user/password credentials
+version: "3.9"
+services:
+  esphome:
+    image: esphome/esphome:2024.4
+    volumes:
+      - ./esp32/config:/config
+      - /etc/localtime:/etc/localtime:ro
+      - /dev/ttyUSB0:/dev/ttyUSB0
+    privileged: true
+
+  db:
+    image: postgres:16.3-alpine
+    shm_size: 128mb
+    volumes:
+      - "./database/data:/var/lib/postgresql/data"
+      - "./database/db.sql:/docker-entrypoint-initdb.d/db.sql"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: memoires-info
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres -d memoires-info"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  migration:
+    build:
+      context: .
+      dockerfile: Dockerfile.Sqitch
```

Comment:
this file has been moved to Dockerfile/ folder

Do we really need 2 docker-compose files?
maybe we can reduce duplication with https://docs.docker.com/compose/multiple-compose-files/

Reply:
oh right I forgot to change the path there but the docker-compose.yml is supposed to be the one for development is a bit outdated, the production one is docker-compose.prod.yml

Reply:
I've looked at the link you provide and for debugging it is better for us to have the docker-compose.yml while for production we want to have the smallest and fastest program so we use the docker-compose.prod.yml wich automatically install dependencies and build the project in a way that only the build files will be keep inside the docker image

Reply:
I am fine with differenciating prod/dev environments.

The problem is that we have redundant code in both docker-compose yml files. And then, the risk of forgetting to update one is big. So we could "include" things that are common to both files.